src/main/java/gregtech/api/gui/widgets/GhostCircuitSlotWidget.java

```diff
@@ -0,0 +1,92 @@
+package gregtech.api.gui.widgets;
+
+import gregtech.api.recipes.ingredients.IntCircuitIngredient;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.inventory.ClickType;
+import net.minecraft.item.ItemStack;
+import net.minecraft.network.PacketBuffer;
+import net.minecraftforge.items.IItemHandlerModifiable;
+
+
+/**
+ * Like a phantom slot, but for real fake items like circuits (and maybe molds later?)
+ */
+public class GhostCircuitSlotWidget extends SlotWidget {
```

Comment:
All other slots with similar functionality are called `Phantom*Widget` is here some specific reason this one is named differently?

Reply:
Phantom is something that can accept an item dragged in my JEI.

Reply:
Change it to a better name